recipes/premake/5.x/conandata.yml

```diff
@@ -1,14 +1,10 @@
 sources:
+  # "5.0.0-beta2":
+  #   url: "https://github.com/premake/premake-core/releases/download/v5.0.0-beta2/premake-5.0.0-beta2-src.zip"
+  #   sha256: "4c1100f5170ae1c3bd1b4fd9458b3b02ae841aefbfc41514887b80996436dee2"
```

Comment:
Please do not add a new version if is has not been explicitly requested

Reply:
Why not? I only left it out because it caused too many additional issues for this PR.